src/app.rs

```diff
@@ -967,7 +1130,7 @@ impl eframe::App for ProfApp {
         eframe::set_value(storage, eframe::APP_KEY, self);
     }
 
-    /// Called each time the UI needs repainting.
+    // Called each time the UI needs repainting.
```

Comment:
A triple `///` is a doc comment. This one is from the original template, so it's not exactly amazing. I think we should either leave the doc comment in place, or we should remove the line entirely.